webhook/packages/ghapp/redirect/redirect.py

```diff
@@ -104,6 +108,8 @@ def process(args: dict[str, str]) -> dict:
         channel_name = channels[0]
     elif repository in _channel_for_repository:
         channel_name = _channel_for_repository[repository]
+    elif repository in _ignored_repositories:
+        pass
```

Comment:
If you just do a `pass`, the following print line right after this code` print(f"Event for repository {repository} and channel {channel_name}")` will throw an `UnboundLocalError`.
Well, it's catched in the `main` function but always throwing errors for ignored repos is weird.

I think `return {"body": "ignored repository"}` is much better here.